clrplus/Scripting.MsBuild/Packaging/ProjectPlus.cs

```diff
@@ -636,7 +636,10 @@ public class ProjectPlus : Project, IDisposable {
             }
 
             return _buildTimePaths.GetOrAdd(condition, () => new ListWithAction<string>(path => {
-                
+                if (!path.EndsWith(";")) {

```

Comment:
Just make sure it ends with a ";". Apparently the "Prefix" attribute won't do that for you for some reason.